src/ripple/core/LoadMonitor.h

```diff
@@ -37,10 +37,6 @@ class LoadMonitor
     explicit
     LoadMonitor (beast::Journal j);
 
-    void addCount ();
```

Comment:
We need to convert `LoadMonitor` over to `<chrono>`.  Not necessarily in this pull request.